scripts/fetch-entity/src/drupalClient.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * This is essentially a copy of what's in the drupalClient.ts module, but
+ * copying it over here for use in the script because importing it from
+ * next-build/src/lib/drupal/drupalClient.ts is a pain.
+ */
+
+import { DrupalClient } from 'next-drupal'
+
+/* eslint-disable import/no-extraneous-dependencies */
+import { SocksProxyAgent } from 'socks-proxy-agent'
+import fetch from 'node-fetch'
+
+const syswideCas = await import('syswide-cas')
+syswideCas.addCAs('certs/VA-Internal-S2-RCA-combined.pem')
+syswideCas.addCAs('certs/rootCA.pem')
+
+const baseUrl =
+  process.env.NEXT_PUBLIC_DRUPAL_BASE_URL || 'https://va-gov-cms.ddev.site'
+
+const debug = process.env.DEBUG === 'true'
+
+const proxyFetch = (input: RequestInfo, init: RequestInit = {}) => {
+  const agent = new SocksProxyAgent('socks5h://127.0.0.1:2001')
+  return fetch(input, { ...init, agent })
+}
```

Comment:
Deliberately not using our `proxy-fetch` package.
1. It was too painful to try to include it; module resolution nightmare
2. We don't actually need any of the bells and whistles with `cross-fetch` or `p-retry` 🤷